ecs-cli/modules/cli/servicediscovery/servicediscovery_helper.go

```diff
@@ -65,8 +65,8 @@ func resolveIntPointerFieldOverride(c *cli.Context, flagName string, ecsParamsVa
 	}
 
 	if flagVal != nil && ecsParamsVal != nil {
-		paramsVal := string(*ecsParamsVal)
-		override := string(*flagVal)
+		paramsVal := strconv.FormatInt(*ecsParamsVal, 10)
+		override := strconv.FormatInt(*flagVal, 10)
```

Comment:
You could also just use `strconv.Itoa` here since that defaults the radix to 10?